src/components/NcCheckboxRadioSwitch/NcCheckboxContent.vue

```diff
@@ -208,7 +208,8 @@ export default {
 	user-select: none;
 	min-height: $clickable-area;
 	border-radius: $clickable-area;
-	padding: 4px $icon-margin;
+	box-sizing: content-box !important;
```

Comment:
Not sure if this is the best solution, we could also set `min-width: $clickable-area;` and set the padding to `padding: 4px 10px`. That would not require an `!important`.